src/olympia/users/notifications.py

```diff
@@ -8,15 +8,6 @@ class _NOTIFICATION(object):
     pass
 
 
-class thanks(_NOTIFICATION):
-    id = 2
-    group = 'user'
-    short = 'dev_thanks'
-    label = _('an add-on developer thanks me for a contribution')
-    mandatory = False
-    default_checked = True
-
-
```

Comment:
Do we still need that somewhere in the UI to read old data?

Reply:
If we wanted to show the (now obsolete) notification we could but I'm not sure what it would give us.  The data is still in the database - I wasn't going to remove it because it didn't matter but I've just realised the API would try to return it if set and fail because the object didn't exist.  So I'll add a migration to drop those db entries.

Reply:
https://github.com/mozilla/addons-server/pull/7011/commits/4794d6748aef8f212d40007d4762b7f32724dbbc